inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/SinkField.java

```diff
@@ -33,10 +33,14 @@
     @ApiModelProperty("Source field type")
     private String sourceFieldType;
 
+    @ApiModelProperty("Is source builtin, 0: no, 1: yes")
+    private Integer isSourceBuiltin = 0;
+
     public SinkField(int index, FieldType fieldType, String fieldName, String fieldComment,
-            String fieldValue, String sourceFieldName, String sourceFieldType) {
+            String fieldValue, String sourceFieldName, String sourceFieldType, Integer isSourceBuiltin) {
```

Comment:
'isSourceBuiltin' is difficult to understand ,  use metaField instead?

Reply:
done